sway/tree/workspace.c

```diff
@@ -708,6 +708,11 @@ void workspace_for_each_container(struct sway_workspace *ws,
 struct sway_container *workspace_find_container(struct sway_workspace *ws,
 		bool (*test)(struct sway_container *con, void *data), void *data) {
 	struct sway_container *result = NULL;
+    if (ws == NULL){
+        sway_log(SWAY_ERROR, "Cannot find container with no workspace.");
+        return NULL;
+    }
```

Comment:
This specific crash is fixed by the change in `view.c`, although I also added this as a safeguard against calling `workspace_find_container` with no workspace.